JetLagged/app/src/main/java/com/example/jetlagged/backgrounds/StripesShaderBackground.kt

```diff
@@ -23,27 +23,23 @@ import androidx.annotation.RequiresApi
 import androidx.compose.animation.core.withInfiniteAnimationFrameMillis
 import androidx.compose.runtime.mutableFloatStateOf
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.draw.drawWithCache
-import androidx.compose.ui.graphics.Brush
 import androidx.compose.ui.graphics.ShaderBrush
 import androidx.compose.ui.graphics.drawscope.ContentDrawScope
 import androidx.compose.ui.node.DrawModifierNode
 import androidx.compose.ui.node.ModifierNodeElement
-import com.example.jetlagged.ui.theme.White
 import com.example.jetlagged.ui.theme.Yellow
-import com.example.jetlagged.ui.theme.YellowVariant
 import kotlinx.coroutines.launch
 import org.intellij.lang.annotations.Language
 
-private data object YellowBackgroundElement : ModifierNodeElement<YellowBackgroundNode>() {
+private data object StripesBackgroundElement : ModifierNodeElement<StripesBackgroundNode>() {
```

Comment:
Can you rename to `MovingStripesBackgroundElement`, just to give more information that there will be movement